sim/warrior/items.go

```diff
@@ -253,3 +253,50 @@ func (warrior *Warrior) ApplyQueensfallWarriorEffect(aura *core.Aura) {
 		},
 	})
 }
+
+// Striking a higher level enemy applies a stack of Coup, increasing their damage taken from your next Execute by 10% per stack, stacking up to 20 times. At 20 stacks, Execute may be cast regardless of the target's health.
+func (warrior *Warrior) ApplyRegicideWarriorEffect(aura *core.Aura) {
+	// Coup debuff array
+	debuffAuras := warrior.NewEnemyAuraArray(func(unit *core.Unit, _ int32) *core.Aura {
+		aura := unit.RegisterAura(core.Aura{
+			ActionID:  core.ActionID{SpellID: 1231417},
```

Comment:
And here's the warrior aura https://www.wowhead.com/classic-ptr/spell=1231416/coup same deal as the others!